agent/multi_agent.py

```
@@ -26,57 +28,49 @@ async def run(self, *args, **kwargs):
+            elif run.status == 'requires_action' and run.required_action:
+                tool_outputs = await self._handle_tool_calls(run.required_action, file_id)
+                
+                try:
+                    run = client.beta.threads.runs.submit_tool_outputs(
+                        thread_id=thread_id,
+                        run_id=run.id,
+                        tool_outputs=tool_outputs
+                    )
+                except Exception as e:
+                    return f"Error submitting tool outputs: {e}"
+                continue # Loop back to check run status
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)